README.md

```diff
@@ -2,10 +2,13 @@
 A web app where you can find contributors for your open-source project or as an individual contributor you can find open-source projects. Best place to start building in public.
 Find people who have the same vision as you do and contribute to each other for building amazing products.
 
+## Connect with us
+<p align='left'>Discord:  &nbsp &nbsp <a href="https://discord.gg/M2BMPdku" target="_blank" rel="noreferrer"> <img src="https://www.svgrepo.com/show/353655/discord-icon.svg" alt="Discord" width="30" height="30" align="top"/> </a></p>
+
```

Comment:
let's remove the discord text, only keep icon as we are facing alignment issue

Reply:
cool